spec/game_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'rspec'
+require_relative '../App/Game/game'
+
+RSpec.describe Game do
+  let(:game) { Game.new(1, nil, nil, nil, '2023-07-01', true, '2023-07-01') }
+
+  describe '#initialize' do
+    it 'sets the id' do
+      expect(game.id).to eq(1)
+    end
+
+    it 'sets the multiplayer attribute' do
+      expect(game.multiplayer).to eq(true)
+    end
+
+    it 'sets the last_played_at attribute' do
+      expect(game.last_played_at).to eq('2023-07-01')
+    end
+  end
+
+  describe '.load_games_data' do
+    it 'loads game data from the file' do
+      expect(File).to receive(:exist?).with('games.json').and_return(true)
+      expect(File).to receive(:empty?).with('games.json').and_return(false)

```

Comment:
- [ ] Dear Team, Just as suggested by the previous reviewer, We need to test methods inside your classes, Here I can see the `can_be_archived` method is not tested among others, Please make sure to test some of the main methods inside your `game` class including the `can_be_archived`.